modules/core/src/main/scala/gem/TCC.scala

```diff
@@ -61,10 +61,11 @@ object Tcc {
     // we repeat them below, namespaced per instrument for type safety.
 
     object F2 {
-      val Fpu       = Key[F2FpUnit   ]("fpu"      )(_.tccValue)
-      val Filter    = Key[F2Filter   ]("filter"   )(_.tccValue)
-      val LyotWheel = Key[F2LyotWheel]("lyotWheel")(_.tccValue)
-      val Disperser = Key[F2Disperser]("disperser")(_.tccValue)
+      val Fpu         = Key[F2FpUnit     ]("fpu"        )(_.tccValue)
+      val Filter      = Key[F2Filter     ]("filter"     )(_.tccValue)
+      val LyotWheel   = Key[F2LyotWheel  ]("lyotWheel"  )(_.tccValue)
+      val Disperser   = Key[F2Disperser  ]("disperser"  )(_.tccValue)
+      val WindowCover = Key[F2WindowCover]("windowCover")(_.tccValue)
```

Comment:
The `tccValue` seems to be an integral part of the definition of all instruments fields.  I found that surprising.  Do we send `lyotWheel` to the TCC today for example?  I followed the pattern and added a `tccValue` for the window cover because it is a fundamental part of importing data from XML.